### DISCUSSION TITLE
How do I get a screenshot of an EMu window?

### DISCUSSION THREAD

**Discussion by reguzik** [fieldmuseum/EMu-Documentation#373] [FAQs] [answered]
How do I get a screenshot of an EMu window?

**Comment by reguzik** [comment, marked as answer]
1) If you want to grab the whole of your desktop. PrtScr

2) If you just want the EMu module window Ctrl-PrtScr

3) If you want the EMu module window AND the menu options Use the Alt keys to open the menu you want to see Ctrl-PrtScr

Then you can use Ctrl-P or paste to insert the image into a document or email.

**Comment by magpiedin** [comment]
On a Mac: 
- **Cmd+Shift+4**  = you can draw a box around an EMu window
- **Cmd+Shift+3** = screenshot the whole desktop

Screenshots will be auto-saved, generally to your desktop as "Screenshot [DATE].png"
